### ISSUE TITLE
Hyperparameters of SuperGLUE finetuning

### ISSUE THREAD

**Issue by richardbaihe** [microsoft/fastformers#7]
Hi,
Thanks for sharing this fantastic repository!

Could you provide the hyperparameters used when finetuning the superglue tasks? 
For example, I noticed the num_train_epochs=10 and learning rate is 1e-5 in README. 
What about the batch_size?
Does every task share the same hyperparameters?

**Comment by ykim362** [comment]
Hi @richardbaihe . Thanks for your interest!

I used more epochs for the smaller datasets such as copa, cb. Some other tasks also benefits from more epochs.
But, ReCoRD's dataset is quite huge, so I usually use only 2 epochs for it. Even 2 epochs take very long time compared to the other tasks.
I used from 7e-6 to 5e-5. usually 1e-5.

For the batch size, I used 16 in most cases. If GPU memory cannot handle, I reduced the batch size.

So, I used different hyperparameters for different tasks.
